docs/components/modeler/forms/configuration/forms-config-options.md

```diff
@@ -0,0 +1,44 @@
+---
+id: forms-config-options
+title: Options Source
+description: How to configure an options source on a form field
+---
+
+## Configuring form field options sources
+
+Certain form fields work from a set of pre-populated options that your user will want to select from, such as the [Radio](../form-element-library/forms-element-library-radio.md) or [Select](../form-element-library/forms-element-library-select.md) fields. This source can be configured several ways, as described below.
+
+### Static :: Configured on the form schema
```

Comment:
@christinaausley what are your thoughts on this? From a technical standpoint, I think this might create some challenges with how Docusaurus creates anchor links, but I'm not sure.

Reply:
Ah, yes. Regardless of Docusaurus' reaction, I really try to avoid any punctuation in headers as I've seen them cause complications in the past.